sources/platform/runtime/machine-image/services-versions.md

```diff
@@ -103,10 +117,13 @@ This page lists the versions of Services available for each supported [OS](/plat
 |3.2.2  | - Ubuntu 14.04<br>- Ubuntu 16.04|[v5.7.3](/platform/runtime/machine-image/ami-v573/)
 |3.1.1  | - Ubuntu 14.04<br>- Ubuntu 16.04|[v5.6.1](/platform/runtime/machine-image/ami-v561/) and earlier
 
+Note: Neo4j is not supported on v6.8.4 images since the images are based on Java 10 and neo4j is not supported on that version of Java.
```

Comment:
Note: Neo4j is not supported on v6.8.4 images because it is incompatible with Java 10.

Reply:
ok